api/src/wfl/api/workloads.clj

```diff
@@ -78,16 +85,15 @@
   "Use transaction `tx` to load `workload`(s) with `project`."
   [tx project]
   (log/debugf "Loading workloads with project=\"%s\"" project)
-  (let [do-load   (partial load-workload-impl tx)]
-    (mapv do-load
-          (jdbc/query tx ["SELECT * FROM workload WHERE project = ?" project]))))
+  (let [query-str "SELECT * FROM workload WHERE project = ? ORDER BY id ASC"]
+    (mapv (partial load-workload-impl tx) (jdbc/query tx [query-str project]))))
```

Comment:
This kinda breaks the formal similarity between the 2 `load-workloads` defns.

Reply:
Don't follow. How what formal similarity is this?

Reply:
Given that you've factored out `query-str` and inlined `do-load` in `load-workloads-with-project`,
you could do the same with `load-workloads` to restore the common structure they used to share.
Not terribly important but it tripped me up while reviewing the file and your changes:
"Why change this but not that?  Now they look more different than they are.  How different are they really?"
YMMV, of course.

Reply:
Maybe …
```clojure
(defn load-workloads
  "Use transaction `tx` to load all known `workloads`."
  [tx]
  (log/debug "Loading all workloads")
  (let [query-str "SELECT * FROM workload ORDER BY id ASC"]
    (mapv (partial load-workload-impl tx) (jdbc/query tx [query-str]))))
```